src/main/java/redis/clients/jedis/mcf/MultiClusterTransaction.java

```diff
@@ -13,7 +13,15 @@
 import java.util.Queue;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import redis.clients.jedis.*;
+import redis.clients.jedis.Builder;
+import redis.clients.jedis.BuilderFactory;
+import redis.clients.jedis.CommandArguments;
+import redis.clients.jedis.CommandObject;
+import redis.clients.jedis.CommandObjects;
+import redis.clients.jedis.Connection;
+import redis.clients.jedis.RedisProtocol;
+import redis.clients.jedis.Response;
+import redis.clients.jedis.TransactionBase;
```

Comment:
For 9 imports, it's okay to use `*` :slightly_smiling_face: 
```suggestion
import redis.clients.jedis.*;
```